molecule/logging-test/molecule.yml

```diff
@@ -0,0 +1,99 @@
+---
```

Comment:
`molecule/logging-test/molecule.yml`, `molecule/monitoring-test/molecule.yml` and `molecule/ops-agent-test/molecule.yml` seem identical. Can they be shared instead?

Reply:
Each test needs its own molecule.yml file so I don't there's a way around having 3 separate files. 
The logging and monitoring molecule.yml files are identical so I created a hard link between them. The ops-agent file is subtly different in that it doesn't test on ubuntu 2010.

Reply:
Thanks for the clarification!